apps/InfiniteRedAI/app/screens/HomeScreen/demoInfo.ts

```diff
@@ -30,4 +30,10 @@ export const DEMO_LIST: DemoInfo[] = [
     screen: "ImageLabeling",
     image: FACE_HOLDER,
   },
+  {
+    title: "Document Scanner",
+    description: "Quickly and easily digitize paper documents on Android",
+    screen: "DocumentScanner",
+    image: FACE_HOLDER,
```

Comment:
Can you take a screenshot or make a preview of some kind? (if not not a biggie)

Reply:
Will do!

Reply:
<img width="406" alt="image" src="https://github.com/infinitered/react-native-mlkit/assets/374022/29b3b738-5d84-4c1d-8042-18d463032a77">